src/blast.ts

```diff
@@ -1,16 +1,40 @@
 import { config } from "./config";
+import { box } from "./util";
 
 export class Blast {
-    constructor(
-        public posX: number,
-        public posY: number,
-        public elem: HTMLElement,
-        public color: string,
-        public opacity: number,
-        public size: number = config.playerSize,
-    ) {
+    public opacity = 1.0;
+    private elem: HTMLDivElement;
+
+    constructor(public posX: number, public posY: number, public color: string, public size: number = config.playerSize) {
+        this.elem = document.createElement("div");
```

Comment:
I feel like having classes keep track of their own elements is probably a good step to take here, since it means less code within our gigantic `script.ts` file (and a much nicer looking `frame` function)